setup.cfg

```diff
@@ -33,9 +32,7 @@ install_requires =
     scipy>=1.7.0
     tqdm>=4.60.0
 
-
 [options.extras_require]
-# complete package functionality
```

Comment:
could we keep the comments explaining what the individual extras_require sections are for?

Reply:
Ah I see! Yeah I'd prefer to keep them, would it be fine like this?
```ini
full = # complete package functionality
    joblib>=1.0.0
    matplotlib>=3.5.0
    mne>=1.0.0
    numba>=0.55.0;python_version<'3.11'
    tensorflow>=2.7.0;python_version<'3.11'
    wfdb>=3.4.0

```

Reply:
Yes, that's fine! I'll add the comments.